setuptools_grpc/__init__.py

```diff
@@ -1,3 +1,10 @@
 """Setuptools grpc."""
+from .build_grpc import build_grpc
 
 __version__ = '0.2'
+
+
+def install(dist):
+    dist.cmdclass.update(build_brpc=build_grpc)
+    build = dist.get_command_obj("build")
+    build.sub_commands = [("build_grpc", None)] + build.sub_commands
```

Comment:
> Setuptools somehow automagically filters subcommands, only the first one is actually run.

OK, that's true. It only runs each command once.

> We could, but it would probably be non-trivial to extract options from commands that hasn't been called yet.

`Distribution.get_command_obj` should take care about that.

Reply:
> I'd move it to a submodule.
> I'm afraid dist.cmdclass.update prevents customization of build_grpc command.

I removed `dist.cmdclass.update` and import. Therefore, there is no need to move the `install` to another module.

> > We could, but it would probably be non-trivial to extract options from commands that hasn't been called yet.

> Distribution.get_command_obj should take care about that.

How? You receive uninitialized object, but you won't have any configuration loaded, not before distribution init is completed. If you know how to resolve that, please present a snippet.

Reply:
> You receive uninitialized object, but you won't have any configuration loaded, not before distribution init is completed.

OK, I found it in the `distutils` code. Now, I'm afraid it will make also the change of the `build` command more difficult. The `get_command_obj('build')` also returns command ignoring all the configuration options and will not get updated further on. That might cause unexpected behavior.

Reply:
We found, it's ofically supported design.